internal/driver/driver.go

```diff
@@ -0,0 +1,147 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2018 Canonical Ltd
+// Copyright (C) 2018-2019 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Package driver this package provides a simple example implementation of
```

Comment:
This isn't an example, it's an GPIO implementation of the `ProtocolDriver` interface.